dev/Insomnia-workspace.yaml

```diff
@@ -17,7 +17,7 @@ resources:
         value: t-1
         description: ""
         id: pair_b733d24c32904ab5a8cc102428f5ea68
-      - name: metricName
+      - name: metricKey
```

Comment:
still metric key...

Reply:
Should I add one more request for meticGroup?